frontend/src/components/NewRunParametersV2.test.tsx

```diff
@@ -281,6 +281,146 @@ describe('NewRunParametersV2', () => {
     });
   });
 
+  it('test convertInput function for LIST type with default value', () => {
+    const handleParameterChangeSpy = jest.fn();
+    const props = {
+      titleMessage: 'default Title',
+      pipelineRoot: 'defalut pipelineRoot',
+      specParameters: {
+        listParam: {
+          parameterType: ParameterType_ParameterTypeEnum.LIST,
+          defaultValue: [1, 2, 3],
+        },
+      },
+      handlePipelineRootChange: jest.fn(),
+      handleParameterChange: handleParameterChangeSpy,
+    };
+    render(<NewRunParametersV2 {...props}></NewRunParametersV2>);
```

Comment:
NIT: We can change to the following for simplicity:

`<NewRunParametersV2 {...props} />`